.gitignore

```diff
@@ -15,3 +15,4 @@ bin/
 
```

Comment:
I'm not sure if you should be including the docs' html files, but I got a shock when I saw that I had to look through 113 files!